File: solutions/cpp/easy/arrays-introduction/solution-approach1.cpp
// ──────────────────────────────────────────────────
// Link        https://www.hackerrank.com/challenges/arrays-introduction/problem?isFullScreen=true
// Problem     Arrays Introduction
// Difficulty  Easy
// Subdomain   Introduction
// Platform    HackerRank
// Language    cpp
// Status      Accepted
// Submitted   2026-07-27, 06:42 p.m.
// ──────────────────────────────────────────────────

#include <cmath>
#include <cstdio>
#include <vector>
#include <iostream>
#include <algorithm>
using namespace std;


int main() {
    /* Enter your code here. Read input from STDIN. Print output to STDOUT */   
    int n;
    int arr[10000];
    cin >> n;
    for(int i = 0; i <= 10000; i++){
        cin >> arr[i];
    }
    std :: reverse(arr, arr+n);
    
    for(int i=0; i < n ; i++){
        cout<<arr[i]<<" ";
    }
    
    return 0;
}

File: solutions/python3/easy/py-set-add/solution-approach1.py
# ──────────────────────────────────────────────────
# Link        https://www.hackerrank.com/challenges/py-set-add/problem?isFullScreen=true
# Problem     Set .add() 
# Difficulty  Easy
# Subdomain   Sets
# Platform    HackerRank
# Language    python3
# Status      Accepted
# Submitted   2026-07-28, 11:04 a.m.
# ──────────────────────────────────────────────────

# Enter your code here. Read input from STDIN. Print output to STDOUT
a = input()
n = set()

for i in range(int(a)):
    n.add(input())
print(len(list(n)))

